### PR TITLE
Match server data extraction to current server schema.

### PR DESCRIPTION
**Pull request by mcomella**
This PR may change. It depends on:
 * What the server schema is in code. Existing PRs: [one][1] [two][2]
 * The data pushed to the server at any point.

[1]: https://github.com/mozilla-mobile/prox-server/pull/8
[2]: https://github.com/mozilla-mobile/prox-server/pull/14/

### CODE REVIEW

**Comment by mcomella** [comment]
[Fetch Places based on location](https://trello.com/c/eCsOQZeb/120-fetch-places-based-on-location)

**Comment by mcomella** [comment]
[Wire up Places with Place Carousel Cards](https://trello.com/c/fDEYEBI3/119-wire-up-places-with-place-carousel-cards)

File: Prox/Prox/Models/Place.swift [hunk 1]
```diff
@@ -35,8 +35,8 @@ class Place {
     init?(fromFirebaseSnapshot data: FIRDataSnapshot) {
         guard data.exists(), data.hasChildren(),
                 let value = data.value as? NSDictionary,
-                let summary = value["description"] as? String ?? value["pullQuote"] as? String,
-                let name = value["id"] as? String, // TODO: change to name from id
+                let summary = value["summary"] as? String,
```
Comment (fluffyemily) [inline Prox/Prox/Models/Place.swift]: I get no places when I run this code as there is no `summary` key, only a `description` key

**Comment by mcomella** [comment]
Unfortunately, I think the server schema changed again. I can update locally for the demo and we should update the client when we know the server-side data won't be changing anymore.

**Comment by mcomella** [comment]
Made [another schema request](https://github.com/mozilla-mobile/prox-server/pull/17) to James: I'll figure this out and get this landed.

**Comment by mcomella** [comment]
Close in favor of #55.

**Pull request closed without merging**